src/gather_stats.ts

```diff
@@ -275,16 +329,27 @@ function calculate_wait(when, opened, reviews) {
     return when.diff(latest_feedback);
 }
 
-function transform_issue_nodes(issue_nodes) {
+type CookedIssueNode = {
+    id: number;
+    opened: DateTime;
+    closed: DateTime;
+    labeled_cxx20: boolean;
+    labeled_cxx23: boolean;
+    labeled_lwg: boolean;
+    labeled_bug: boolean;
+};
+
+function transform_issue_nodes(issue_nodes: RawIssueNode[]): CookedIssueNode[] {

```

Comment:
The reason is that this builds up an array of objects without ever saying that it's an array of `CookedIssueNode`:

https://github.com/microsoft/STL/blob/81f02e863977f249645ec7f0e0b0b80bc4040473/src/gather_stats.ts#L351-L359

TypeScript will deduce that this returns an array of stuff, but not that it returns `CookedIssueNode[]`, which is needed by later functions.

This could also be solved by creating a `ret` variable with the type `CookedIssueNode` and then returning that - I'm not sure which is more preferred, but the explicit return type seemed more readable.